packages/frontend/src/components/TableTree.tsx

```diff
@@ -236,12 +236,19 @@ const TableTree: FC<TableTreeProps> = ({
                                 isDimension: false,
                             } as NodeDataProps,
                             isSelected: selectedNodes.has(fieldId(metric)),
-                            secondaryLabel: metric.source && (
-                                <NodeItemButtons
-                                    node={metric}
-                                    onOpenSourceDialog={onOpenSourceDialog}
-                                />
-                            ),
+                            secondaryLabel:
+                                (metric.isAutoGenerated && (
+                                    <Icon
+                                        style={{ width: '25px' }}
```

Comment:
```suggestion
                                        style={{ marginRight: 7 }}
```
This makes it more aligned